modules/backend/widgets/Filter.php

```diff
@@ -145,18 +145,19 @@ public function renderScopeElement($scope)
                 break;
 
             case 'numberrange':
-                if ($scope->value && is_array($scope->value) && count($scope->value) === 2 &&
-                    $scope->value[0] &&
-                    $scope->value[1]
+                if (
+                    $scope->value
+                    && (is_array($scope->value) && count($scope->value) === 2)
```

Comment:
Do you really need brackets here @bennothommo or it's just for readability purpose?

Reply:
@w20k Just for readability. When it's all on one line without brackets, it's quite hard to discern what is happening :)